lib/std/fs/File.zig

```diff
@@ -1149,13 +1149,23 @@ pub fn readToEndAllocOptions(
     comptime alignment: u29,
     comptime optional_sentinel: ?u8,
 ) !(if (optional_sentinel) |s| [:s]align(alignment) u8 else []align(alignment) u8) {
-    // If no size hint is provided fall back to the size=0 code path
-    const size = size_hint orelse 0;
+    const size = undefined;
```

Comment:
I think you meant to use `var` here. Do run tests locally to catch things like this (will be a compile error).

Reply:
Fixed it.
Ran tests locally and they pass : ) thanks